ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json

```diff
@@ -310,7 +310,7 @@
           "[EPSMANAGING]",
           "[LAMANAGING]"
         ],
-        "CRUD": "CR"
+        "CRUD": "R"
```

Comment:
Do you want to move all of the things relating to this event to their relevant deprecated folders?